persistence/src/main/resources/db/migration/data/integration/V1_17_1__Update_StateHistories_Position.sql

```diff
@@ -0,0 +1 @@
+UPDATE states_history SET position_id = 1 WHERE state_id = 4;
```

Comment:
3 StateHistories added to "Candidate C" during the integration don't have positions attached to them. Thence, the header doesn't work properly for it, only for the newly created applications.

It's not an essential change, though.